.github/workflows/pythonbuild.yml

```diff
@@ -1,10 +1,15 @@
 name: Build
 
+# Schedule runs to run twice a day
+
+
 on:
   push:
     branches:
       - master
   pull_request:
+  schedule:
+      - cron: '0 8 * * *' # This schedule runs at 08:00 UTC every day
```

Comment:
can we run it at 5 when we're sleeping 😄

Reply:
oh, this is UTC. So it should be 1pm UTC. Let me change it.